packages/grid_client/scripts/algorand.ts

```diff
@@ -1,102 +1,142 @@
+  BlockchainGetModel,
+  BlockchainSignModel,
+  GridClient,
+} from "../src";
 import { getClient } from "./client_loader";
 import { log } from "./utils";
+
 let grid3: GridClient;
 const test_account_address = "EBABIFYHKVSMFEV66M7SMIILEZCJ4SHBC3FPGISFFTRIIG6VX3KGSAXR34";
 const mnemonic =
   "wood clump category carpet cabin cement joy cover this hour armor twist write trade term only label later lizard disease boil pelican dish ability this";
 const message = "a message";
+const receiver = "HZ57J3K46JIJXILONBBZOHX6BKPXEM2VVXNRFSUED6DKFD5ZD24PMJ3MVA";
 const hexMessage = message
   .split("")
   .map((letter, i) => message.charCodeAt(i).toString(16))
   .join("");
+const name = "testAcc";
+
 async function main() {
+  const account: AlgorandAccountCreateModel = {
+    name: "myaccount",
+  };
+  const importedAccount: AlgorandAccountInitModel = {
+    name: name,
+    secret: mnemonic,
+  };
+  const accountAddress: AlgorandAccountAssetsFromAddressModel = {
+    address: test_account_address,
+  };
+  const accountName: BlockchainGetModel = {
+    name: name,
+  };
+  const transaction: AlgorandCreateTransactionModel = {
+    name: name,
+    address_dest: receiver,
+    amount: 100,
+    description: "test",
+  };
+  const deleteAccount1: BlockchainDeleteModel = {
+    name: name,
+  };
+  const deleteAccount2: BlockchainDeleteModel = {
+    name: "myaccount",
+  };
+  const message: BlockchainSignModel = {
+    content: hexMessage,
+    name: name,
+  };
+  const transactionModel: AlgorandCreateTransactionModel = {
+    name: name,
+    address_dest: receiver,
+    amount: 100,
+    description: "test",
+  };
+  const transfer: AlgorandTransferModel = {
+    name: name,
+    address_dest: receiver,
+    amount: 100,
+    description: "test",
+  };
+
   await getClient().then(res => (grid3 = res));
-  await createAccount("myaccount");
-  await importAccount("manana", mnemonic);
-  await getAccountAssetsFromAddress(test_account_address);
-  await getAccountAssetsFromName("manana");
-  await createTransaction("manana");
-  await signBytes("manana", hexMessage);
-  await signTransaction("manana");
-  await getAccountMnemonicFromName("manana");
-  await checkAccountExistsFromName("manana");
+  await createAccount(account);
+  await importAccount(importedAccount);
+  await getAccountAssetsFromAddress(accountAddress);
+  await getAccountAssetsFromName(accountName);
+  await createTransaction(transaction);
+  await signBytes(message);
+  await signTransaction(transactionModel);
+  await getAccountMnemonicFromName(accountName);
+  await checkAccountExistsFromName(accountName);
   await listAccounts();
-  await transferAccount("manana");
-  await deleteAccount("manana");
-  await deleteAccount("myaccount");
+  await transferAccount(transfer);
+  await deleteAccount(deleteAccount1);
+  await deleteAccount(deleteAccount2);
 
   grid3.disconnect();
 }
 
-async function createAccount(account_name: string) {
+async function createAccount(account) {
```

Comment:
why there're no comments here in all functions like the rest of scripts?

Reply:
added in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1409/commits/b899322cc957a8cb0501bb3c8ce49f0ed59969c5